jax/_src/profiler.py

```diff
@@ -96,6 +98,10 @@ def start_trace(log_dir, create_perfetto_link: bool = False):
     create_perfetto_link: A boolean which, if true, creates and prints link to
       the Perfetto trace viewer UI (https://ui.perfetto.dev). The program will
       block until the link is opened and Perfetto loads the trace.
+    create_perfetto_trace: A boolean which, if true, additionally dumps a
+      ``perfetto_trace.json.gz`` file that is compatible for upload with the
+      Perfetto trace viewer UI (https://ui.perfetto.dev). The file will also be
```

Comment:
Maybe add that this is useful if you wanna generate the trace without blocking execution.

Reply:
Done!